### ISSUE TITLE
NTR: Trauma to external ear canal (Ears)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7816]
New term request Trauma to external ear canal (segal_210901150635): History of using cotton ear swabs included  UMLS: 2186634

**Comment by pnrobinson** [comment]
@MickeySegal
Can you provide context. qtip usage is not universally considered trauma...

**Comment by MickeySegal** [comment]
We have this listed for Otitis Externa.  Although Q-Tips are often used gently, sometimes they damage the skin.  There is some recent research about the nose: https://www.sciencealert.com/research-in-mice-suggests-a-surprising-link-between-nose-picking-and-alzheimers

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000774
- label: History of recent trauma to external ear canal
- parents: [Recent past medical history (HP:4000144)](https://hpo.jax.org/app/browse/term/HP:4000144)
- pmid: 
- def: Medical history of trauma to the external ear canal, including cotton-swab use, in recent weeks.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7816
- synonyms:
